crates/re_log_types/src/time.rs

```diff
@@ -62,15 +73,30 @@ impl Time {
         self.nanos_since_epoch() % (24 * 60 * 60 * 1_000_000_000) == 0
     }
 
+    // TODO(paris): Is there a pattern in Rust where you put free functions in an anonymous namespace versus in the struct?
+    // TODO(paris): Should we #[inline] this? I'm guessing the suggestion is to benchmark with and without, though this perhaps seems a bit big for inlining.
+    fn time_string(
+        datetime: OffsetDateTime,
+        parsed_format: &Vec<FormatItem<'_>>,
+        time_zone_for_timestamps: TimeZone,
+    ) -> String {
+        let local_offset =
+            (time_zone_for_timestamps == TimeZone::Local).then(UtcOffset::current_local_offset);
```

Comment:
a better pattern here would be to do a `match` over the timezone settings. That way it stops compiling when somebody adds a new enum value but doesn't handle it here :)

```rust
match time_zone_for_timestamp {
    TimeZone::Local => { ... }
     TimeZone::Utc => { ... }
}
```

Reply:
Yep good call, updated. Leads to some repeat of code now, but seems worth it to avoid future issues.